packages/amplify-e2e-tests/src/rds-v2-test-utils/index.ts

```diff
@@ -365,12 +371,36 @@ export const checkOperationResult = (result: any, expected: any, resultSetName:
     delete item['__typename'];
     expect(item).toEqual(expected[index]);
   });
+
+  if (errors && errors.length > 0) {
+    expect(result.errors).toBeDefined();
+    expect(result.errors).toHaveLength(errors.length);
+    result.errors?.forEach((error: any, index: number) => {
```

Comment:
If order doesn't matter, would it be less brittle to use `arrayContaining` matcher?

Reply:
That's a good point. Order shouldn't matter here much. Updated.